recipes/default.rb

```diff
@@ -17,7 +17,7 @@
 # limitations under the License.
 #
 package node['rsyslog']['package_name']
-package rsyslog_relp_package if node['rsyslog']['use_relp']
+package rsyslog_relp_package if node['rsyslog']['use_relp'] unless platform_family?('freebsd')
```

Comment:
Oops, I meant this :)
```suggestion
package rsyslog_relp_package if node['rsyslog']['use_relp'] && unless platform_family?('freebsd')
```